100-Resources/200-Contributing/200-Contribute-to-Docs.mdx

```diff
@@ -21,15 +21,16 @@ Xata's developer documentation follows a [docs-as-code](https://www.writethedocs
 
 1.  **Fork repository:** Fork the [`mdx-docs`](https://github.com/xataio/mdx-docs) repository on GitHub to create your copy.
 2.  **Clone repository:** Clone your forked repository to your local machine using Git. Navigate to the cloned `mdx-docs` repository and add the upstream repository as a remote using `git remote add upstream <upstream-repo-url>`.
-3.  **Create branch:** Create a new branch for your changes using `git checkout -b <my-docs-update>`. You can create any branch name, but including the issue number or a brief description is recommended.
-4.  **Edit documentation:** Make necessary changes to documentation files, considering Markdown and MDX syntax.
-5.  **Commit changes:** Add and commit your changes using `git add .` and `git commit -m "Update documentation"` with an informative commit message.
-6.  **Push to fork:** Push changes to your forked repository with `git push origin my-docs-update`.
-7.  **Create pull request:** Visit the `mdx-docs` repository on GitHub, and create a new pull request from your branch.
-8.  **Provide details:** Add a clear title and description for your PR, explaining the purpose of your changes.
-9.  **Submit PR:** Submit the pull request, and it will be reviewed by the Xata Docs team.
-10. **Collaborate:** Collaborate during the review process, make requested changes, and work towards PR approval.
-11. **Approval and merge:** Once your PR meets all requirements and is approved by the team, it will be merged into the main branch, and your changes will become part of the official Xata documentation.
+3.  **Install linter:** Run `pnpm install` to lint your code and align with style standards. If this is not done your pull request will fail some checks.
```

Comment:
This is optional, the bot will now fix any lint issues

Reply:
@SferaDev I think, the last contributor issue failed in the repo https://github.com/xataio/mdx-docs/pull/252. I just assumed it was because the linter did not run?